packages/clangml/clangml.4.4.0/opam

```diff
@@ -10,7 +10,7 @@ homepage: "https://memcad.gitlabpages.inria.fr/clangml/"
 doc: "https://memcad.gitlabpages.inria.fr/clangml/doc/clangml/index.html"
 bug-reports: "https://gitlab.inria.fr/memcad/clangml/issues"
 depends: [
-  "conf-libclang"
+  "conf-libclang" {= "12"}
```

Comment:
Is there a reason why this is now required here? Does it not work with 10 or 11 ?

Reply:
ah nevermind I just saw the new configure option. Mmh, the annoying thing is that this change would break people with lock files with both clangml.4.4.0 and conf-libclang.11. Could this be in a new revision `clangml.4.4.0-1`?

Reply:
I just made a try with `<= 12`: I hope that if the variable `conf-libclang:clangml440_configure_options` is not defined, it will be replaced by the empty string, which will in turn be ignored by `./configure` (with just a warning).

Reply:
Indeed, failures and successes are the same with `<=`!